### ISSUE TITLE
Radiative surface optics blend unclipped sea-ice fraction: emissivity reaches 1.9, polar albedo 0.95

### ISSUE THREAD

**Issue by duncanwp** [climate-analytics-lab/jax-gcm#703]
## What is wrong

`_surface_optical_properties` in `jcm/physics/forcing/echam_boundary_conditions.py:67-90` blends land / ocean / sea-ice emissivity and albedo by tile fraction, but **never clips the sea-ice fraction against the land fraction**:

```python
ocean_fraction = jnp.maximum(1.0 - land_fraction - sea_ice_fraction, 0.0)
```

Over polar land where the forcing bundle has `icec = 1` *and* `lsm = 1`, the partition becomes land = 1, ice = 1, ocean = 0 — the fractions **sum to 2**, and the blend is no longer a convex combination:

```
emissivity = 1.0*0.95 (land) + 0.0*0.98 (ocean) + 1.0*0.95 (sea ice) = 1.90
```

## Impact

The value reaches the solver unclamped. `radiation.surface_emissivity` → `jcm/physics/radiation/rrtmgp.py:1262` → `:1319` → `:859` `sfc_emis=`, with no clip anywhere on the path. RRTMGP is therefore told **ε = 1.9** over Antarctica and the Arctic: surface longwave emission is ~90% too large and the surface reflectance `1 - ε` is **negative** (−0.9).

The **same bug corrupts the shortwave albedo**, which went unnoticed because its values are not impossible on their face: `radiation.surface_albedo_vis` and `_nir` both max at **0.95** (= land 0.15 + ice 0.80, and 0.25 + 0.70). Antarctic visible albedo should be ~0.80.

Measured in a T63L47 ECHAM+RRTMGP run: tile fractions sum to 2.0 at **3855 of 18432** grid points; 3672 points carry emissivity > 1. The affected points are polar land — 1863 south of 60S (max at lat −88.6), 1176 north of 60N, 631 in 30–60N.

This is an instantaneous-field bug, not an output-diagnostic artifact: feeding the real `bundles/t63/terrain.nc` `lsm` and `forcing_pd.nc` `icec` directly into `_surface_optical_properties` reproduces the saved field (min/max 0.950/1.900, 3647 points > 1, correlation 0.99968 against the model output).

## Where the hooks already are

The clip already exists in the *other* two consumers of `sice_am`, added by #334 ("Surface fraction setup missing ice and sea ice integration"); the radiative optics term was missed:

- `jcm/physics/surface/echam/surface_physics.py:425-428` — `sea_ice_fraction = jnp.clip(forcing.sice_am, 0.0, 1.0 - land_fraction)`
- `jcm/physics/vertical_diffusion/tte_tke/vertical_diffusion.py:587-589` — identical

So today **the ECHAM surface tiles and the radiation see different tile partitions**, which is its own consistency problem.

## Decisions for whoever picks this up

1. **Which convention.** The one-line clip above is minimal and matches the rest of `jcm`. ECHAM's own convention instead stores sea ice as a fraction *of the ocean part* of the box (grid-box ice = `(1 - slf) * seaice`), which agrees over full land but differs on partial-coast cells. Pick one deliberately and say why in the comment.
2. **The input bundle.** `icec = 1` over the Antarctic interior is arguably a bundle defect (permanent land ice flagged as sea ice). The clip makes the model robust either way, but the bundle may want fixing too.
3. **Surface/radiation ε disagreement (secondary).** The ECHAM surface tiles compute their own longwave emission with `params.emissivity` defaulting to **0.99** for all three tiles (`jcm/physics/surface/echam/surface_types.py:49`, used in `land.py:429`, `ocean.py:165`, `sea_ice.py:196`), while radiation uses the 0.95/0.98/0.95 blend. The surface energy budget and the radiation code do not agree on ε even where the fractions are sane. Worth resolving in the same pass or splitting out.

## How it was found

While validating RRTMGP training data for the radiation NN emulator (#702). The emulator's offline generator now clips emissivity into [0, 1] on input (`tools/radiation_emulator/generate_training_data.py`, `INPUT_BOUNDS`) so the training labels are not poisoned; that clip is a data-generation guard and is **not** a fix for the model path.
